### ISSUE TITLE
Cannot drag Tasks panel into secondary side bar

### ISSUE THREAD

**Issue by kburtram** [microsoft/azuredatastudio#21007]
- Azure Data Studio Version: `insiders`

Steps to Reproduce:

1. Open Tasks panel and Secondary Side Bar
2. Attempt to move Tasks to Secondary Side Bar

Expected: Tasks can dock in side bar similar to other panels

Actual: Tasks isn't dockable in side bar

![image](https://user-images.githubusercontent.com/599935/198194697-69c36d92-d524-477b-9b8a-e64961d0690f.png)

**Comment by alexepperly** [comment]
Yeah, the secondary side bar is basically useless for me. I wanted to have Connections open on the right and the Explorer open on the left, but the secondary side bar is essentially inert.

**Comment by CaidenKehrer** [comment]
Seconding this one. It is still an issue in latest azure data studio versions.
